### ISSUE TITLE
Why is the FS of bcftools mpileup always 0?

### ISSUE THREAD

**Issue by robertzeibich** [samtools/bcftools#1834]
Why is the FS of bcftools mpileup always 0?

running version 1.15

**Comment by pd3** [comment]
Good question, thank you for opening the issue. This was added by https://github.com/samtools/bcftools/commit/8d790c8035ea26bbd139104fa9a2eab6820686fc but the value is not filled anywhere in the code. Not sure if was because a piece of code got lost during the merge of a complex pull request or if it was never implemented, probably just added with the intention to do it later and forgotten.

**Comment by pd3** [comment]
This has been now fixed, with two changes
- the p-value is reported, not phred-scaled
- the annotation is not added by default

**Comment by robertzeibich** [comment]
Thx. When will that be available?

**Comment by pd3** [comment]
It already is, from github http://samtools.github.io/bcftools/howtos/install.html
Don't know when the next versioned release will be yet.

**Comment by paigeduffin** [comment]
Hello, I'm revisiting this comment as I've run into a similar observation in my dataset- the FS field (fisher test for strand bias) does not seem to be correctly populated — it is coming up as zero for all genotypes so everything is passing. I've read through this thread and am a bit confused- is this something that needs to be explicitly specified (ie, accounting for this is not the default)? 

We've just spent a long time calling reads for a very large dataset and would like to avoid doing it all over again. Is there a way to populate this field after the bcftools mpileup? We have retained the read depth for for/rev reads supporting alt/ref so is it possible to calculate this without recalling genotypes? Thank you so much!
